src/Adapter/LegacyHookSubscriber.php

```diff
@@ -249,11 +249,10 @@ public static function getSubscribedEvents()
                         $functionName = 'call_' . $id . '_' . $moduleId;
                         $moduleListeners[] = array($functionName, 2000 - $order);
                     }
-
-                    if (count($moduleListeners)) {
-                        $listeners[$name] = $moduleListeners;
-                    }
+                } else {
```

Comment:
A bug fix on Hook Collector: some hooks are not dispatched if no modules register it... and the MAIN interest of the collector is to be able to list every hook, it's like the best doc we can have because it's always up to date :+1: